src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -32,13 +32,15 @@ class Connection
         'group' => 'symfony',
         'consumer' => 'consumer',
         'auto_setup' => true,
+        'xadd_maxlength' => 0, // 0 disables trimming, any other value defines an approximate maximum length
```

Comment:
I think `xadd` should stay an internal implementation detail. Also, as per https://github.com/antirez/redis/blob/unstable/redis.conf#L1310 it may also be worth mentioning that this option is about limiting the number of entries and not bytes. 

Could `stream_max_entries` be a better name?

Reply:
True. I've renamed the config :)